config/globals.js

```diff
@@ -181,4 +181,8 @@ module.exports = function () {
       })
     }
   });
+
+  const organisationHelper = require(ROOT_PATH+"/module/organisations/helper");
```

Comment:
@aman-tunerlabs - I am thinking, should we set it from the list helper function, so whenever the first time list is called, it checks in session, if yes, return from there, else make API call, set in session the same data and return from there. Let me know